### ISSUE TITLE
Problem with RB 30 and TD MX

### ISSUE THREAD

**Issue by ACCURA68** [FrSkyRC/ETHOS-Feedback-Community#3005]
I have a simple question to see if you can help me. About the TD MX receiver... How many channels does it deliver at most via SBUS OUT...? 16 or 24? I am configuring an RB 30 with two TDMX as primary and secondary, in SBUS 16 both the receivers and the RB30 work correctly but if I try to activate the 24 channels, sbus 24, I get the channels to work by moving servos but in the RB30 it won't even let me enter its menu, nor in the fail safe nor can I map channels... Do you know what I may be doing wrong or is it a limitation of the TDMX to only 16 channels?
Greetings from Spain!

**Comment by mawzthefinn** [comment]
Are you using an X20 Pro?

If so, try setting your channel range to 1-21 and see if it works then. This came up elsewhere.

**Comment by ACCURA68** [comment]
> Are you using an X20 Pro?
> 
> If so, try setting your channel range to 1-21 and see if it works then. This came up elsewhere.

Yes, I´m using an X20 Pro.  I will try set only 1-21 channels, lets see and comment you. Thanks you a lot.

**Comment by ACCURA68** [comment]
> Are you using an X20 Pro?
> 
> If so, try setting your channel range to 1-21 and see if it works then. This came up elsewhere.

Hello! I just checked following your instructions and I have to say... IT WORKS!!! Assigning channels from 1 to 24 in RF system and putting both primary TD mx and secondary TD mx receivers in SBUS 24... IT WORKS and lets me access all the RB30 menus. Of course, I had to deactivate the telemetry of the secondary TD mx because it gave me sensor loss errors, with the telemetry in only one it works perfectly. Tremendously grateful but... How did you arrive at that solution?
